### ISSUE TITLE
Normalization of texts for search indexing

### ISSUE THREAD

**Issue by mora145** [CleverRaven/Cataclysm-DDA#58755]
### Is your feature request related to a problem? Please describe.

Hi guys. I am an English>Spanish translator

I'm having trouble in the text filter, the 'accented' words are a problem for someone who doesn't usually write with accents. 
Is there a way to 'normalize' the search text for the filter, so the accents could be removed before starting the search. 

_Example
Galón > Galon._
Carbón > Carbon._

In most of the text names Spanish speakers don't usually look for things using accents. But the translation of all items has been done using them. The filter is strict when searching the string, so if you forget to put the accent, it will not find anything.

### Solution you would like.

Player and vendor items should be normalized before the search. That way it won't matter if the text is accented or not.

### Additional context

Spanish translation

**Comment by BrettDong** [comment]
Duplicate of #56938.
